MetaMask/metamask-extension
# Update UI for blocked tokens, use the latest dev gas api URL

## Explanation
This PR:
- Updates UI for blocked tokens in Swaps. Previously, we removed all blocked tokens from a list of tokens shown to a user, except when it was a token where they already had some non-zero balance. When they tried to swap a blocked token with  non-zero balance, they got a message: "No Quotes Available", since we blocked the quote request on backend. 
We still hide blocked tokens on the frontend with this change, except when a user searches for a blocked one or they have a non-zero balance for it. In these cases we show the token in the list, but it's disabled and unable to be selected for a swap.
- Uses the latest dev gas api URL

## Manual Testing Steps
Scenario 1: 0 balance for a blocked token
- Click on "Swap from" in Swaps
- Scroll and look for a blocked token (e.g. LUNC if you are in the US). It will not be in the list by default. 
- Type LUNC in the search box, it will appear, by it will be disabled and you can't select it for a swap. On mouse hover it should say: "Token is not available to swap in this region"

Scenario 2: non-zero balance for a blocked token
- If you have a non-zero balance for a blocked token, it should be visible in the list, but disabled and you can't select it for a swap. On mouse hover it should say: "Token is not available to swap in this region"
